source/extensions/filters/http/gcp_authn/gcp_authn_filter.h

```diff
@@ -32,7 +32,7 @@ struct GcpAuthnFilterStats {
   ALL_GCP_AUTHN_FILTER_STATS(GENERATE_COUNTER_STRUCT)
 };
 
-using FilterConfigProtoSharedPtr =
+using FilterConfigSharedPtr =
```

Comment:
Pleas make the type std::shared_ptr<**const** envoy::extensions::filters::http::gcp_authn::v3::GcpAuthnFilterConfig>; to indicate that config is immutable.

Reply:
Sure Done!